.github/workflows/node.js.yml

```diff
@@ -0,0 +1,94 @@
+name: Build & Deploy
+
+on:
+  push:
+    branches:
+      - staging
+      - master
+  pull_request:
+    branches:
+      - staging
+      - master
+  workflow_dispatch:
+
```

Comment:
include this `concurrency` section: https://github.com/CruGlobal/godtools-android/blob/develop/.github/workflows/build.yml#L9-L11

it basically means that only 1 action can run at a time and it will cancel the previous if it hasn't completed yet